#
# Data Filter Configuration
#
# custom filter_config_t
	float32 cutoff_frequency  # cutoff frequency, normalized by filter rate
	bool enabled  # RESERVED (1 = filter enabled | 0 = filter disabled)
